sentencepiece/ubuntu/Rakefile

```diff
@@ -0,0 +1,52 @@
+require "rake"
+require "octokit"
+require "open-uri"
+
+UBUNTU_CODE_NAMES = "xenial,bionic,disco"
+UBUNTU_VERSIONS = "16.04,18.04,19.04"
+PACKAGE = "sentencepiece"
+LAUNCHPAD_PPA = "groonga-ppa"
+
+client = Octokit::Client.new
+releases = client.releases("google/sentencepiece")
+current_release = releases.find do |release|
+  release.tag_name
+end
+version = current_release.tag_name.sub(/v/, "")
+archive_name = "#{PACKAGE}-#{version}-Source.tar.xz"
+
+desc "Download SentencePiece source packages"
+task :download do
+  download_url_base = "https://github.com/google/sentencepiece/releases/download"
+  download_url = "#{download_url_base}/#{current_release.tag_name}/#{archive_name}"
+  puts "Download #{download_url}"
+  open(download_url) do |downloaded_file|
+    open("#{archive_name}", "wb") do |output_file|
+      output_file.print(downloaded_file.read)
+    end
+  end
+end
+
+desc "Upload SentencePiece source packages"
+task :upload do
+  upload_script = ENV["UPLOAD_SCRIPT_PATH"]
+  unless upload_script
+    puts "Set UPLOAD_SCRIPT_PATH (e.g. $HOME/work/groonga/groonga/packages/ubuntu/upload.rb)"
+    exit 1
```

Comment:
You should use `raise` instead of `puts` and `exit` in Rakefile: https://github.com/red-data-tools/packages.red-data-tools.org/blob/master/repository-task.rb#L16

Reply:
Thanks, I've fixed.